trs_filer/config.py

```diff
@@ -0,0 +1,34 @@
+import os
```

Comment:
See comment at @sarthakgupta072's PR regarding this: Gunicorn config should ideally go into FOCA. Perhaps you can take care of that, as @sarthakgupta072 has already added some other recent change to FOCA that will be good for you as well (merging specs)?